desk/app/contacts.hoon

```diff
@@ -238,11 +238,10 @@
     ::
     ++  p-init
       |=  wen=(unit @da)
+      ::  TODO: figure out if we can just give the profile every time
       ?~  wen  (give (fact ~ full+rof))
       ?:  =(u.wen wen.rof)  cor
-      ::
-      :: no future subs
-      ?>((lth u.wen wen.rof) (give (fact ~ full+rof)))
+      (give (fact ~ full+rof))
```

Comment:
Isn't the thing you actually want ("give the profile if it's changed since the date they asked about") more like this?

```suggestion
      ?:  (gte u.wen wen.rof)  cor
      (give (fact ~ full+rof))
```